### PR TITLE
Update points system 

### PR DESCRIPTION
**Pull request by imenelydiaker**
A proposition for the points system.

### CODE REVIEW

File: docs/mmteb/readme.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The contribution points are computed using the following table:
 
 | Contribution type   | Demand              | Points  | Description                                                                                                       |
 | ------------------- | ------------------- | ------- | ----------------------------------------------------------------------------------------------------------------- |
-| New dataset         | As many as possible | 2+bonus | The first dataset for a language x task gains 4 bonus points. |
+| New dataset         | As many as possible | 2+bonus | The first dataset for a language x task gains 4 bonus points. If the number of new languages is >= 12 then contributors gains 50 points (12 * 4 + 2 = 48 + 2 = 50).|
```
Comment (isaac-chung) [inline docs/mmteb/readme.md:51]: wdyt?
```suggestion
| New dataset         | As many as possible | 2+bonus | The first dataset for a language x task gains 4 bonus points. If the number of new languages is >= 12 then points for that PR for a new dataset are capped at 50 (12 * 4 + 2 = 48 + 2 = 50).|
```

**Review by isaac-chung — APPROVED** [review]

**Comment by KennethEnevoldsen** [comment]
Following the discussion at https://github.com/embeddings-benchmark/mteb/pull/636#discussion_r1597467581

Should we also change the >100 points to 100 otherwise I believe this is ready to be merged in.

File: docs/mmteb/validate_points.py [hunk 2]
```diff
@@ -5,6 +5,17 @@
...
 
+commit_exceptions = {
+    "scores_from_old_system",
+    # <100 points: from before max points were enforced
+    "440",
+    "543",
+    "616",
+    "636",
+    # >100 points: from before max points were enforced (reduced to 100 points)
+    "583",
```
Comment (KennethEnevoldsen) [inline docs/mmteb/validate_points.py:16]: In agreement with author

Comment (KennethEnevoldsen) [reply docs/mmteb/validate_points.py:16]: @imenelydiaker if we are to be rigid here I believe we should also reduce the point of flores. I don't believe it changes the scores of any from the team to less than 10 (still fairly high).

**Comment by KennethEnevoldsen** [comment]
@imenelydiaker and @isaac-chung will merge this in to avoid new PRs getting merged in with a too-high score.

**Pull request merged**